### PR TITLE
A11yfixes

### PR DESCRIPTION
**Pull request by GurjeevKaur**
Focussed State for all fields and error messages with error icon .

### CODE REVIEW

File: packages/Checkbox/Checkbox.jsx [hunk 1]
```diff
@@ -184,6 +212,7 @@ CheckboxWithRef.defaultProps = {
   name: '',
   value: true,
   styles: {},
+  keyboardAccessibility: function () {},
```
Comment (mgauthier-nascent) [inline packages/Checkbox/Checkbox.jsx:215]: () => {} should be favoured, but it's not 'wrong' :)

File: packages/Input/Input.jsx [hunk 2]
```diff
@@ -215,13 +254,17 @@ export const Input = ({
       setDisplay(!display)
     }
   }
-
+  console.log('disply' + display)
```
Comment (mgauthier-nascent) [inline packages/Input/Input.jsx:257]: search and remove all console.logs in pm-kit before pushing changes

Comment (GurjeevKaur) [reply packages/Input/Input.jsx]: Removed from code.

File: packages/Checkbox/Checkbox.jsx (hunk 1 above)
Comment (GurjeevKaur) [reply packages/Checkbox/Checkbox.jsx]: Removed from code.

**Review by mgauthier-nascent — APPROVED** [review]

**Pull request merged**